components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -44,55 +44,55 @@ const getCategory = makeGetCategory();
 const taskLabels = {
     [OnboardingTasksName.CHANNELS_TOUR]: (
         <FormattedMessage
-            id='onboardingTask.checklist.task_channels_tour'
-            defaultMessage='Take a tour of channels'
+            id='onboardingTask.checklist.task_learn_more_about_messaging'
+            defaultMessage='🔀 Learn about messaging'
         />
     ),
     [OnboardingTasksName.BOARDS_TOUR]: (
         <FormattedMessage
-            id='onboardingTask.checklist.task_boards_tour'
-            defaultMessage='Manage tasks with your first board'
+            id='onboardingTask.checklist.plan_sprint_with_kanban_style_boards'
+            defaultMessage='🎯 Plan a sprint with Kanban-style boards'
         />),
     [OnboardingTasksName.PLAYBOOKS_TOUR]: (
         <FormattedMessage
-            id='onboardingTask.checklist.task_playbooks_tour'
-            defaultMessage='Explore workflows with your first Playbook'
+            id='onboardingTask.checklist.task_resolve_incidents_faster_with_playbooks'
+            defaultMessage='🐛 Resolve incidents faster with playbooks'
         />
     ),
     [OnboardingTasksName.INVITE_PEOPLE]: (
         <FormattedMessage
-            id='onboardingTask.checklist.task_invite'
-            defaultMessage='Invite team members to the workspace'
+            id='onboardingTask.checklist.task_invite_people_to_team'
+            defaultMessage='👋 Enhance your Mattermost experience. Invite your team to check out our developer platform'
```

Comment:
@pvev heads up that we only want to change the first top three items, everything from this one down should only add the emoji and no text change cc / @annelieseklein